### ISSUE TITLE
Spanner clients hangs for a few seconds after running a query

### ISSUE THREAD

**Issue by gsouf** [googleapis/nodejs-spanner#1324]
#### Environment details

  - OS: linux
  - Node.js version: 10
  - npm version: 7
  - `@google-cloud/spanner` version: 5.5.0

#### Steps to reproduce

  1. Create the following javascript file in `test.js`
  
```js
const Spanner = require("@google-cloud/spanner").Spanner;

const projectId = "projectid";
const instance = "instance";
const database = "database";
process.env.SPANNER_EMULATOR_HOST = "localhost:9010";

const spanner = new Spanner({
    projectId: projectId,
});
const spannerInstance = spanner.instance(instance);
const spannerDatabase = spannerInstance.database(database, {
    acquireTimeout: 5000,
});

async function run() {
    try {
        console.time("run and close");
        await spannerDatabase.run({
            sql: `SELECT 1`,
        });

        await spannerDatabase.close();
        console.timeEnd("run and close");
    } catch (e) {
        console.error("caught", e);
    }
}

run();
```

  2. run `time node test.js`

Output:

```
$ time node ./test.js 
run and close: 142.309ms

real    0m5.420s
user    0m0.575s
sys     0m0.066s
```

As you can see it hangs after the query has run and it takes 5 seconds to finish the process.

The same without a the sql query `spannerDatabase.run` will finish almost immediately.

I only tried with spanner emulator.

Thanks

**Comment by skuruppu** [comment]
@olavloite if you could please take a look at this one, that would be awesome.

**Comment by olavloite** [comment]
This problem is caused by the `acquireTimeout: 5000` setting. This will cause the client library to use a 5 second timeout whenever a session is acquired from the pool. The timeout handler is however not cleared when a session is returned to the application.

The workaround in version 5.6.0 and earlier is to not set an `acquireTimeout`. A fix is offered in #1327

**Comment by gsouf** [comment]
thank you @olavloite @skuruppu 

Is that going to be released with version 5.7 ?

**Comment by olavloite** [comment]
> Is that going to be released with version 5.7 ? When can we expect it to be bumped?

It should be included in 5.6.1 and higher: https://github.com/googleapis/nodejs-spanner/pull/1328

**Comment by skuruppu** [comment]
I will merge the release PR shortly. Hopefully it'll be done by tomorrow.
